setup.py

```diff
@@ -208,13 +208,12 @@ def get_git_sha() -> str:
         "starrocks": ["starrocks>=1.0.0"],
         "doris": ["pydoris>=1.0.0, <2.0.0"],
     },
-    python_requires="~=3.9",
+    python_requires="~=3.10",
     author="Apache Software Foundation",
     author_email="dev@superset.apache.org",
     url="https://superset.apache.org/",
     download_url="https://www.apache.org/dist/superset/" + version_string,
     classifiers=[
-        "Programming Language :: Python :: 3.9",
```

Comment:
ok so we want to keep this line for the time being

Reply:
@michael-s-molina Do we want to wait until version 4.0 is released? And after this, will we merge this PR?

Reply:
We don't have any dependency with 4.0 to merge this PR. If you address both comments from @mistercrunch, we'll be able to merge it.

Reply:
@mistercrunch Can you push the changes and merge it? Because I don't have time for it now